core/runtime/src/main/java/org/eclipse/sw360/antenna/configuration/ConfigurationReader.java

```diff
@@ -93,7 +96,9 @@ private File resolveConfigUriAndGetFile(URI configFileUri, Path antennaTargetDir
                     LOGGER.warn("Destination file already existed, continuing by overwriting the file.");
                 }
                 LOGGER.debug("Copy configuration file to target folder of antenna.");
-                org.apache.commons.io.FileUtils.copyURLToFile(configFileUri.toURL(), configFromUri);
+                try (InputStream stream = configFileUri.toURL().openStream()) {
+                    Files.copy(stream, configFromUri.toPath(), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
`configFromUri` file (and thus the parent directory) exists due to `createNewFile` in line 94.